src/atlas/array/helpers/ArrayForEach.h

```diff
@@ -286,90 +286,107 @@ struct ArrayForEach {
+            typename Function,
+            typename = std::enable_if_t<
+                execution::is_execution_policy<ExecutionPolicy>()>>
+  static void apply(ExecutionPolicy, std::tuple<ArrayView...>&& arrayViews,
+                    const Mask& mask, const Function& function) {
+    detail::ArrayForEachImpl<ExecutionPolicy, 0, ItrDims...>::apply(
+        std::move(arrayViews), mask, function, std::make_tuple(),
+        std::make_tuple());
   }
 
   /// brief   Apply "For-Each" method
   ///
   /// details Apply ForEach with default execution policy.
   template <typename... ArrayView, typename Mask, typename Function>
-  static void apply(std::tuple<ArrayView...>&& arrayViews, const Mask& mask, const Function& function) {
-      apply(std::move(arrayViews), mask, function);
+  static void apply(std::tuple<ArrayView...>&& arrayViews, const Mask& mask,
+                    const Function& function) {
+    apply(std::move(arrayViews), mask, function);
   }
 
   /// brief   Apply "For-Each" method
   ///
-  /// details Apply ForEach with run-time determined execution policy and no mask.
+  /// details Apply ForEach with run-time determined execution policy and no
+  /// mask.
   template <typename... ArrayView, typename Function>
-  static void apply(const eckit::Parametrisation& conf, std::tuple<ArrayView...>&& arrayViews, const Function& function) {
+  static void apply(const eckit::Parametrisation& conf,
+                    std::tuple<ArrayView...>&& arrayViews,
+                    const Function& function) {
     apply(conf, std::move(arrayViews), detail::no_mask, function);
   }
 
   /// brief   Apply "For-Each" method
   ///
-  /// details Apply ForEach with compile-time determined execution policy and no mask.
+  /// details Apply ForEach with compile-time determined execution policy and no
+  /// mask.
   template <typename ExecutionPolicy, typename... ArrayView, typename Function,
-            typename = std::enable_if_t<execution::is_execution_policy<ExecutionPolicy>()>>
-  static void apply(ExecutionPolicy executionPolicy, std::tuple<ArrayView...>&& arrayViews, const Function& function) {
+            typename = std::enable_if_t<
+                execution::is_execution_policy<ExecutionPolicy>()>>
+  static void apply(ExecutionPolicy executionPolicy,
+                    std::tuple<ArrayView...>&& arrayViews,
+                    const Function& function) {
     apply(executionPolicy, std::move(arrayViews), detail::no_mask, function);
   }
 
   /// brief   Apply "For-Each" method
   ///
   /// details Apply ForEach with default execution policy and no mask.
   template <typename... ArrayView, typename Function>
-  static void apply(std::tuple<ArrayView...>&& arrayViews, const Function& function) {
+  static void apply(std::tuple<ArrayView...>&& arrayViews,
+                    const Function& function) {
     apply(execution::seq, std::move(arrayViews), function);
   }
-
 };
 
```

Comment:
It looks like clang-format has inadvertently touched this file. The only material change is between lines 382 and 388 and `#include <utility>`.